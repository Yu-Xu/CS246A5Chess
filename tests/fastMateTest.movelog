8 rnbqkbnr
7 pppppppp
6  _ _ _ _
5 _ _ _ _ 
4  _ _ _ _
3 _ _ _ _ 
2 PPPPPPPP
1 RNBQKBNR
  abcdefgh
8 rnbqkbnr
7 pppppppp
6  _ _ _ _
5 _ _ _ _ 
4  _ _ P _
3 _ _ _ _ 
2 PPPPP_PP
1 RNBQKBNR
  abcdefgh
8 rnbqkbnr
7 pppp_ppp
6  _ _ _ _
5 _ _ p _ 
4  _ _ P _
3 _ _ _ _ 
2 PPPPP_PP
1 RNBQKBNR
  abcdefgh
8 rnbqkbnr
7 pppp_ppp
6  _ _ _ _
5 _ _ p _ 
4  _ _ PP_
3 _ _ _ _ 
2 PPPPP_ P
1 RNBQKBNR
  abcdefgh
White is in check
8 rnb_kbnr
7 pppp_ppp
6  _ _ _ _
5 _ _ p _ 
4  _ _ PPq
3 _ _ _ _ 
2 PPPPP_ P
1 RNBQKBNR
  abcdefgh
