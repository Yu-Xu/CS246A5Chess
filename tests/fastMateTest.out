8 rnb_kbnr
7 pppp_ppp
6  _ _ _ _
5 _ _ p _ 
4  _ _ PPq
3 _ _ _ _ 
2 PPPPP_ P
1 RNBQKBNR
